index.bs

```diff
@@ -4773,15 +4775,7 @@ enum IDBTransactionMode {
 
 <div algorithm>
 
-The <dfn attribute for=IDBTransaction>objectStoreNames</dfn> getter steps are:
-
-1. If [=/this=] is an [=/upgrade transaction=],
-    then return a new {{DOMStringList}} associated with a [=sorted name list=] of the [=object-store/names=]
-    of the [=/object stores=] in [=/this=]'s [=/connection=]'s [=object store set=].
-
-1. Otherwise, return a new {{DOMStringList}} associated with a [=sorted name list=] of the
-    [=object-store/names=] of the [=/object stores=] in
-    [=/this=]'s [=transaction/scope=].
+The <dfn attribute for=IDBTransaction>objectStoreNames</dfn> getter steps are to return a new {{DOMStringList}} associated with a [=sorted name list=] of the [=object-store/names=] of the [=/object stores=] in [=/this=]'s [=transaction/scope=].
```

Comment:
This is simpler now, since an update transaction's scope is already the connection's object store set.